bindings/core/tests/secrets_manager.rs

```diff
@@ -17,7 +17,7 @@ async fn generate_addresses() -> Result<()> {
     let response = secret_manager.call_method(method).await;
     match response {
         Response::GeneratedAddresses(addresses) => assert_eq!(
-            addresses[0],
+            addresses[0].to_string(),
```

Comment:
Can remove

Reply:
Let me know if you want me to raise them all. Otherwise just resolve and I won't do the rest.